clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2273,10 +2272,7 @@ void createNewStoreVersion(
   private synchronized VeniceWriter<KafkaKey, byte[], byte[]> getVeniceWriter(PushJobSetting pushJobSetting) {
     if (veniceWriter == null) {
       // Initialize VeniceWriter, TODO: passing configurable PubSubProducerAdapter
```

Comment:
nit: Is this line still valid after your change?

Reply:
Good catch. Thanks